### PR TITLE
Pull_request template

### LINKED ISSUE #1592: Create a PR template

### PR DESCRIPTION
**Pull request by alex-semenyuk**
Close #1592

Based on https://github.com/hyperledger-labs/documentation-template/blob/main/.github/pull_request_template.md

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<!-- to mark a point done, place an x in square brackets. eg [x]-->
+<!-- - [x] done with this task-->
+<!----Please delete options that are not relevant. And in order to tick the check box just but x inside them for example [x] like this----->
+
+- [ ] I have read the contributing guidelines.
+- [ ] I have performed a self-review of my own code or work.
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] My changes generates no new warnings.
+- [ ] My Pull Request title is in format <code>< issue name ></code> eg <code>Added links in the documentation</code>.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
```
Comment (EnriqueL8) [inline .github/pull_request_template.md:34]: Can you add a checkbox about code coverage here please

**Review by EnriqueL8 — COMMENTED** [review]
Thanks - one small comment

File: .github/pull_request_template.md (hunk 1 above)
Comment (alex-semenyuk) [reply .github/pull_request_template.md:34]: added

**Review by EnriqueL8 — APPROVED** [review]
LGTM

**Pull request merged**